reference-service/src/main/resources/db/migration/common/V3.68__change_funding_type_field_name.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE `FundingType`
+CHANGE COLUMN `academic` `allowDetails` bit;
```

Comment:
Previously we set a default value 0 for this column, if we don't add it here, the default value setting will be lost.
Do you think adding a default value would be better?

Reply:
Yes good spot. I have changed it. Thanks